test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -1210,9 +1210,52 @@ UCP_INSTANTIATE_TEST_CASE(test_ucp_tag_stats)
 
 class multi_rail_max : public test_ucp_tag_xfer {
 public:
+    enum {
+        VARIANT_64_PATHS  = 100, /* start after parent's enum values */
+        VARIANT_128_PATHS = 101
+    };
+
+    static void get_test_variants(std::vector<ucp_test_variant>& variants)
+    {
+        add_variant_with_value(variants, get_ctx_params(),
+                               VARIANT_64_PATHS, "64_paths");
+        add_variant_with_value(variants, get_ctx_params(),
+                               VARIANT_128_PATHS, "128_paths");
+    }
+
     void init() override
     {
+        /* TODO: 
```

Comment:
can you explain the problem that made us disable RC here?

Reply:
I think we need to implement 2-step wireup for such case like we have for client-server all-devices wireup flow with limited private data space:
1) connect only AM lane
2) all the rest